tests/unittests/tests-nanocoap/tests-nanocoap.c

```diff
@@ -22,12 +23,15 @@
 #include "unittests-constants.h"
 #include "tests-nanocoap.h"
 
+
+#define _BUF_SIZE (128)
```

Comment:
might be good to define as `(128U)`

Reply:
Thanks for pointing this out because I'm unclear about it. When is it necessary/correct to specify unsigned for macros?

Reply:
I think it's always good to be as specific as possible. Also I think it might avoid problems when compiling on different platforms, i.e. 8, 16 or 32 bit.